include/sys/proc.h

```diff
@@ -183,6 +187,8 @@ int proc_getsid(pid_t pid, sid_t *sidp);
  * The SID is returned in `*sidp`. */
 int proc_getsid(pid_t pid, sid_t *sidp);
 
+int proc_getargv(proc_t *p, char *buf, size_t maxlen);
+
```

Comment:
Is this needed?

Reply:
No. It is trash left from previous changes. Removed